### PR TITLE
Create international-programs.php

### PR DESCRIPTION
**Pull request by cdavis-ubc**
Hi
Not sure you got my last message a few minutes ago.  I have updated pep-resources.php in my fork by following your notes (thanks btw :) Let me know if you get notice of those changes or if I have to do something further to get the updated file over to you. Or do I create a new pull request...?

I'll name branches accordingly from now on.  

Thanks again for all your help with this!

### CODE REVIEW

File: templates/education/international-programs.php [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+	<div class="boxey <?php echo esc_attr( $odd_even ); ?> <?php echo esc_attr( $plain_term_slug ); ?> boxISO <?php echo esc_html( $session_type ); ?>">
+
+		<div class="boxey-inside">
+        
+			<?php echo esc_attr( $plain_term_slug ); ?> <span class="sessionType"><?php echo esc_html( $session_type ); ?></span>
+				
+                <div class="boxey-inner">
+                
+					<h3 class="boxTitle"><span class="iso-title"><a title="<?php echo esc_html( $the_title ); ?>" href="<?php the_permalink(); ?>" target="_blank"><?php echo esc_html( $the_title ); ?></a></span></h3>
```
Comment (richardtape) [inline templates/education/international-programs.php]: When you're putting the title in a `title` attribute, use `the_title_attribute()` (as mentioned in my previous message as point 1.)

**Comment by richardtape** [comment]
Looks like I get notifications.

I just replied with a comment about `the_title_attribute()` but other 
than that, looks good.

R

> cdavis-ubc mailto:notifications@github.com
> September 23, 2015 at 8:36 AMvia Postbox 
> https://www.postbox-inc.com/?utm_source=email&utm_medium=sumlink&utm_campaign=reach
> 
> Hi
> Not sure you got my last message a few minutes ago. I have updated 
> pep-resources.php in my fork by following your notes (thanks btw :) 
> Let me know if you get notice of those changes or if I have to do 
> something further to get the updated file over to you. Or do I create 
> a new pull request...?
> 
> I'll name branches accordingly from now on.
> 
> Thanks again for all your help with this!
> 
> ---
> 
> ```
>     You can view, comment on, or merge this pull request online at:
> ```
> 
> https://github.com/ubc/ubc-use-template-shortcode/pull/2
> 
> ```
>     Commit Summary
> ```
> - Create international-programs.php
>   
>   ```
>   File Changes
>   ```
> - _A_ templates/education/international-programs.php
>   https://github.com/ubc/ubc-use-template-shortcode/pull/2/files#diff-0
>   (52)
>   
>   ```
>   Patch Links:
>   ```
> - https://github.com/ubc/ubc-use-template-shortcode/pull/2.patch
> - https://github.com/ubc/ubc-use-template-shortcode/pull/2.diff
> 
> —
> Reply to this email directly or view it on GitHub 
> https://github.com/ubc/ubc-use-template-shortcode/pull/2.

## 

Sent from Postbox 
https://www.postbox-inc.com/?utm_source=email&utm_medium=siglink&utm_campaign=reach

File: templates/education/international-programs.php (hunk 1 above)
Comment (cdavis-ubc) [inline templates/education/international-programs.php]: will do - thanks!

File: templates/education/international-programs.php [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+		<div class="boxey-inside">
+        
+			<?php echo esc_attr( $plain_term_slug ); ?> <span class="sessionType"><?php echo esc_html( $session_type ); ?></span>
+				
+                <div class="boxey-inner">
+                
+					<h3 class="boxTitle"><span class="iso-title"><a title="<?php the_title_attribute(); ?>" href="<?php the_permalink(); ?>" target="_blank"><?php echo esc_html( $the_title ); ?></a></span></h3>
+                    
+					<div class="iso-description"><?php echo wp_kses_post( get_the_excerpt() ); ?> <span class="iso-title hidden"><?php echo esc_html( $the_title ); ?></span></div>
```
Comment (richardtape) [inline templates/education/international-programs.php:44]: When you use `get_the_excerpt()` you don't get any of WordPress's native formatting applied. So you'll just get plain text. Is that what you want? If so, cool. If not, you'll need to `apply_filters( 'the_excerpt', get_the_excerpt() )` in order to get the formatting, should any be added. Won't merge this until I know you're certain.

Comment (cdavis-ubc) [inline templates/education/international-programs.php:44]: The excerpt as plain text is what I want for this template. 
Thanks for checking!

**Comment by richardtape** [comment]
OK merged and on Verf.

**Pull request merged**